lib/sierra-models/base.js

```diff
@@ -253,17 +257,17 @@ class SierraBase {
    *
    * @return {VarFieldMatch[]}
    */
-  varFieldsMulti (marcObjects) {
+  varFieldsMulti (marcObjects, includeMarc = false) {
```

Comment:
I was debating between adding `includeMarc` as a mapping option on the bib mapping, let me know if there is maybe a better place to introduce this flag.

Reply:
As discussed, this makes sense to me now.